addon-test-support/ember-qunit/test-isolation-validation.js

```diff
@@ -19,7 +45,13 @@ const TESTS_NOT_ISOLATED = [];
  */
 export function detectIfTestNotIsolated({ module, name }) {
   if (!isSettled()) {
```

Comment:
We chatted about this a bit, just mentioning it here (should probably be a todo):

We want to essentially do a `await waitUntil(isSettled, { time-out: 2000 })` to reduce some unwanted noise when we enable this by default.